core/modules/installer/installer.theme.inc

```diff
@@ -175,18 +175,87 @@ function template_preprocess_installer_browser_project(&$variables) {
 
   // Check if the project is installed.
   $found = backdrop_get_filename($project['type'], $project['name']);
+
+  // If the project is already downloaded...
   if (_installer_browser_is_project_enabled($project['type'], $project['name'])) {
-    $variables['status'] = '<div class="install-disabled">Already enabled</div>';
-    $variables['install'] = '';
+
+    // ...set the action text and link based on project type.
+    switch ($project['type']) {
+      case 'module':
+        $all_modules_info = system_rebuild_module_data();
+        $project_info = $all_modules_info[$project['name']];
+        // If the module has a coniguration page defined in its .info file.
+        if (isset($project_info->info['configure'])) {
+          $action = 'configure';
+          $url = $project_info->info['configure'];
+        }
+        break;
+      case 'theme':
+        // If the theme has settings.
+        if (backdrop_theme_access($project['name']) && theme_has_settings($project['name'])) {
+          $action = 'settings';
+          $url = 'admin/appearance/settings/' . $project['name'];
+        }
+        break;
+      case 'layout':
+        // All layout themplates can be enabled/disabled in the "Layout
+        // settings" page.
+        $action = 'settings';
+        $url = 'admin/structure/layouts/settings';
+        break;
+    }
+
+    $status = '<div class="install-disabled">' . t('Already enabled');
```

Comment:
Minor, but I think a span may be more appropriate here than a div?

Reply:
Done.